src/java/edu/ucsd/library/dams/api/DAMSAPIServlet.java

```diff
@@ -4279,6 +4279,11 @@ public void output( DAMSObject obj, boolean export,
 			{
 				content = obj.getNTriples(export);
 				contentType = "text/plain";
+			} 
+			else if ( format.equals("turtle") )
+			{
+				content = obj.getTurtle(export);
+				contentType = "text/plain";
```

Comment:
Thanks, Esme. I just corrected it to use text/turtle (https://github.com/ucsdlib/damsrepo/commit/e30fb0b6f79d60605636d29aa5292ef9ed64b056).